### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by WelverinRaven**
I've added a short sentence to every Headline and provided a link to the docker getting started page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,60 @@
...
 ## Motivation
 Besides avoiding to copy it from the last microservice I've written, a friend asked how I'd do it.
 That's is how I'm doing it right now
 
 
+## Download
+
+Clone the repository using 
+
+### SSH
```
Comment (AndersonQ) [inline README.md]: I believe the git part can be omitted. I believe it's ok to assume whoever is seen this repo know how to clone it

File: README.md [hunk 2]
```diff
@@ -1,27 +1,60 @@
...
+
+### https
+
+```bash
+git clone https://github.com/AndersonQ/go-skeleton.git
+```
+or using 
+
+```bash
+go get -u github.com/AndersonQ/go-skeleton.git
```
Comment (AndersonQ) [inline README.md]: 👍 
you need to remove the `.git` at the end
```suggestion
go get -u github.com/AndersonQ/go-skeleton
```

also you could add `go install github.com/AndersonQ/go-skeleton` as well

File: README.md [hunk 3]
```diff
@@ -35,12 +68,18 @@ make build
...
+
+For further information and help around docker visit: [getting started with docker](https://docs.docker.com/get-started/)
+
 ```bash
 docker build -t go-skeleton .
 ```
 
 ## Test
 
+To run a test use the following command:
```
Comment (AndersonQ) [inline README.md]: `make test` runs all tests, so I'd suggest
```suggestion
To run the tests use the following command:
```

File: README.md [hunk 4]
```diff
@@ -1,27 +1,60 @@
 go-skeleton
 -----------
-A skeleton for go microservices
+This project is used as boilerplate example for the structure of a go microservice. A template to get you started so to speak.
+
+A skeleton for go microservices was the core idea, so that's where the name comes from.
```
Comment (AndersonQ) [inline README.md:5]: 👍

File: README.md (hunk 1 above)
Comment (WelverinRaven) [reply README.md]: Yes that's true, but it fills the section with life. Just putting the command there makes it look empty. And if you help even 1 person with the commands, that would already be enough.
But if you want, I'll remove the cloning section.

Comment (AndersonQ) [reply README.md]: I'd like to remove the cloning section. I understand your point, but I believe it creates more noise than value.

Comment (WelverinRaven) [reply README.md]: :+1:

File: README.md [hunk 5]
```diff
@@ -35,12 +53,18 @@ make build
...
+
+For further information and help around docker visit: [getting started with docker](https://docs.docker.com/get-started/)
+
 ```bash
 docker build -t go-skeleton .
 ```
 
 ## Test
 
+To run a test use the following command:
```
Comment (AndersonQ) [inline README.md]:
```suggestion
To run the tests use the following command:
```
I guess you reverted this line by mistake

File: README.md [hunk 6]
```diff
@@ -1,27 +1,45 @@
...
 ## Motivation
 Besides avoiding to copy it from the last microservice I've written, a friend asked how I'd do it.
 That's is how I'm doing it right now
 
 
+## Download
+
+Clone the repository or use 
+```bash
+go get -u github.com/AndersonQ/go-skeleton.git
```
Comment (AndersonQ) [inline README.md]: please remove the `.git` I guess you added back by mistake
```suggestion
go get -u github.com/AndersonQ/go-skeleton
```

Comment (WelverinRaven) [reply README.md]: sorry about that

**Comment by AndersonQ** [comment]
I'm sorry but I'll close this PR. We can discuss better in a issue what would be a nice addition for the readme. Thanks!

**Pull request closed without merging**